src/components/Navbar/LanguageSelector/LanguageSelector.module.scss

```diff
@@ -1,16 +1,16 @@
 .LanguageSelector {
   display: flex;
+  margin-left: 10px;
 
   & .WorldIcon {
-    margin-right: 10px;
```

Comment:
Nice. Boo margin-right lol

Reply:
Well, I will see if I can change it to padding. Margin is not your favourite!